### ISSUE TITLE
"Building a gRPC service with Node.js" "go run client.go" causes "undefined: grpc.SupportPackageIsVersion3"

### ISSUE THREAD

**Issue by jab** [googlecodelabs/cloud-grpc#5]
I just started the "Building a gRPC service with Node.js" codelab and got up to the "go run client.go" step, but it fails with
```
# _/Users/<user>/src/cloud-grpc/start/books
books/books.pb.go:107: undefined: grpc.SupportPackageIsVersion3
```

I'm new to this stack but based on a quick search it looks like grpc/grpc-go#1002 is the issue?

**Comment by jab** [comment]
In case it helps, the URL of the page I'm on is https://codelabs.developers.google.com/codelabs/cloud-grpc/index.html?index=..%2F..%2Findex#3

**Comment by MakMukhi** [comment]
https://github.com/googlecodelabs/cloud-grpc/issues/6

**Comment by jab** [comment]
For anyone else who hits this in the future (until it's fixed in #6), you can work around it with the following patch (generated against latest master, 053463bd50b60f4ec52ebe0f28c0e985c023cb65):
```diff
diff --git a/start/books/books.pb.go b/start/books/books.pb.go
index bb210f0..59d1762 100644
--- a/start/books/books.pb.go
+++ b/start/books/books.pb.go
@@ -104,7 +104,7 @@ var _ grpc.ClientConn

 // This is a compile-time assertion to ensure that this generated file
 // is compatible with the grpc package it is being compiled against.
-const _ = grpc.SupportPackageIsVersion3
+const _ = grpc.SupportPackageIsVersion4

 // Client API for BookService service

@@ -327,7 +327,7 @@ var _BookService_serviceDesc = grpc.ServiceDesc{
 			ServerStreams: true,
 		},
 	},
-	Metadata: fileDescriptor0,
+	Metadata: "books.proto",
 }

 func init() { proto.RegisterFile("books.proto", fileDescriptor0) }
```
Thanks to @MakMukhi for the workaround.
